File: Conectt/Data/Entities/Cliente.cs
using System;

namespace Conectt.Data.Entities
{
    public class Cliente
    {
        public Guid Id { get; set; }
        public string Nome { get; set; }
        public string Cpf { get; set; }
        public DateTime DataNascimento { get; set; }
        public string Email { get; set; }
        public string Empresa { get; set; }
        public string TelefoneComercialDdd { get; set; }
        public string TelefoneComercial { get; set; }
        public string CelularDdd { get; set; }
        public string Celular { get; set; }
        public int Idade { get; set; }
    }
}

File: Conectt/wwwroot/js/vue/cliente.js
var cliente = new Vue({
    el: '#appCliente',
    data: {
        nome: null,
        cpf: null,
        dataNascimento: null,
        email: null,
        empresa: null,
        telefoneComercialDdd: null,
        telefoneComercial: null,
        celularDdd: null,
        celular: null
    },
    methods: {
        
    },
    updated: function () {
        $('#dataNascimento').mask('00/00/0000');
    }
});


//$(document).ready(function () {
    
//    //https://igorescobar.github.io/jQuery-Mask-Plugin/
//});